test/e2e/tests/settings.spec.ts

```diff
@@ -0,0 +1,304 @@
+    await expect(settingsPage.refreshLinkBtn).toBeEnabled();
+    await settingsPage.refreshLinkBtn.click();
+    await settingsPage.confirmRefreshCancelBtn.click();
+  });
+
+  test('invite codes available', async ({ page }) => {
+    await expect(settingsPage.inviteCodesHeader).toBeVisible({ timeout: TIMEOUT_30_SECONDS }); // generous time in case a large list
+    // verify invite codes are available; note on local dev environment they won't be any; we need a pause here because
+    // sometimes even when invite codes header is visible, loading the list of actual invite codes takes a few seconds
+    await page.waitForTimeout(TIMEOUT_3_SECONDS);
+    if (APPT_TARGET_ENV == 'dev') {
+      await expect(settingsPage.noInviteCodesCell).toBeVisible();
+    } else {
+      await expect(settingsPage.noInviteCodesCell).not.toBeVisible();
+      // read the first invite code in the list
+      const firstCode = await settingsPage.firstInviteCode.textContent();
+      expect(firstCode).toHaveLength(36); // uuid is 36 char str including hyphens
+    }
+  });
+
+  test('able to download account data', async ({ page }) => {
+    // setup listener for browser download event
+    const downloadPromise = page.waitForEvent('download');
+    // click the account settings => download your data button and confirm
+    await settingsPage.downloadAccountData();
+    // now verify the browser download did happen successfully
+    const download = await downloadPromise;
+    const downloadedPath = await download.path(); // waits for download to finish
+    console.log(`account data file downloaded to ${downloadedPath}`);
+    expect(downloadedPath).toBeTruthy();
+    expect(await download.failure()).toBeFalsy();
+  });
+
+  test('delete account button', async ({ page }) => {
+    // verify that clicking the delete account button results in a delete account confirm dialog
+    // we won't actually delete the account :) as that would break the other E2E tests
+    await expect(settingsPage.deleteAcctBtn).toBeEnabled();
+    await settingsPage.deleteAcctBtn.click();
+    await settingsPage.confirmDeleteAcctBtn.click();
+  });
+});
+
+test.describe('connected accounts settings', {
+  tag: [PLAYWRIGHT_TAG_E2E_SUITE],
+}, () => {
+  test.beforeEach(async ({ page }) => {
+    // navigate to and sign into appointment
+    await navigateToAppointmentAndSignIn(page);
+    settingsPage = new SettingsPage(page);
+    dashboardPage = new DashboardPage(page);
+    // navigate to the connected accounts settings page
+    await settingsPage.gotoConnectedAccountsSettingsPage();
+  });
+
+  test('edit profile button', async ({ page }) => {
+    // verify that clicking the `edit profile` button redirects to the Mozilla Account profile page
+    await settingsPage.editProfileBtn.click();
+    // on dev env the button doesn't do anything (as expected) but on stage redirects to fxa profile
+    if (APPT_TARGET_ENV !== 'dev') {
```

Comment:
Oh this relies on having this env set:
```
VITE_FXA_EDIT_PROFILE=https://accounts.stage.mozaws.net/settings
```

Reply:
Thanks, good to know! I still prefer to leave this check out when running locally since FxA stage may not necessarily be setup to work with the local dev env.